src/App.js

```diff
@@ -9,7 +10,10 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <div>
+    <h1>React Article</h1>
+    <Article {...article} />
+  </div>
```

Comment:
```suggestion
    <Article {...article} />
```
Оставь только функциональный компонент `<Article {...article} />`.

Reply:
done